tests/js/tests/yii.test.js

```diff
@@ -739,6 +739,7 @@ describe('yii', function () {
     describe('getQueryParams method', function () {
         withData({
             'no query parameters': ['/posts/index', {}],
+            'no query parameters with question char': ['/posts/index?', {}],
```

Comment:
Ideally this needs to be tested for `yii.gridView.js` too. See https://github.com/yiisoft/yii2/blob/9fa66b23a53e8268b1808977146056b92ac60a8a/tests/js/tests/yii.gridView.test.js#L305.